apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx

```diff
@@ -265,6 +265,7 @@ export const EditMultiFactorAuthenticator: FunctionComponent<EditMultiFactorAuth
             });
         }
 
+        /**
```

Comment:
Is there a value in commenting this?

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a